vendor/javascript/importmap-sortablejs.js

```
@@ -0,0 +1,136 @@
+// sortablejs@1.15.0 downloaded from https://ga.jspm.io/npm:sortablejs@1.15.0/modular/sortable.esm.js
+
+/**!
+ * Sortable 1.15.0
+ * @author	RubaXa   <trash@rubaxa.org>
+ * @author	owenm    <owen23355@gmail.com>
+ * @license MIT
+ */
```

ESLint

[error] Expected exception block, space or tab after '/**' in comment.

(spaced-comment)